### ISSUE TITLE
Difficult identify feature class from the legend

### ISSUE THREAD

**Issue by NT-dft** [acteng/atip#490]
It is difficult to recognise which colour the lines representing polution levels are on the map with reference to the legend. Solution could be to change the colour scheme or make the features clickable which would bring up an information box

![image](https://github.com/acteng/atip/assets/135853112/113c15cf-a072-43e7-8e47-647975a367df)
![Capture](https://github.com/acteng/atip/assets/135853112/60e1e6cd-45f4-49c8-a238-014dd69e37fd)

**Comment by dabreegster** [comment]
Agreed. Related to #368 and notes from #378. We're using a raster layer from DEFRA and have no control over the styling, and haven't attempted the API for looking up info at a point. It'd be better for us to find the raw data somewhere and process it into our own layer, so we can control the colour scheme, make the thresholds match inspections requirements, etc.

**Comment by Pete-Y-CS** [comment]
Just to add that this has been raised by someone else: 

> Another ATIP suggestion for your list 1  - this time for Roadside NOx key  - could the colours key be more distinct - I can't tell if the road by Harrogate station is 41-60 or 61-80 as the colours very similar on key .
![image](https://github.com/acteng/atip/assets/133757676/c6a3ddde-b54a-4238-b607-892dd3fa9526)
